jenkins/manila-tempest/10-configure-nfs-server.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -xue
+
+if [[ ! ${USE_SCALITY_IMPL:-} ]]; then
+    USE_SCALITY_IMPL=1
+    echo "Using ${USE_SCALITY_IMPL} as default value for 'USE_SCALITY_IMPL'"
+fi
+
+if [ $USE_SCALITY_IMPL ]; then
+
+    if [[ ! ${RING_VERSION:-} ]]; then
+        RING_VERSION=4
+        echo "Using ${RING_VERSION} as default value for 'RING_VERSION'"
+    fi
+
+    OS_CI_REPOSITORY=https://github.com/scality/openstack-ci-scripts.git
+    NODE=$JCLOUDS_IPS
+    SCRIPT_FILE=$(mktemp)
+
+    # Accept host key
+    ssh-keyscan $NODE >> ~/.ssh/known_hosts
+
+    # Start stuff
+    cat > $SCRIPT_FILE << 'EOF'
+SUP_ADMIN_LOGIN="myName"
+SUP_ADMIN_PASS="myPass"
+INTERNAL_MGMT_LOGIN="super"
+INTERNAL_MGMT_PASS="adminPass"
+HOST_IP=$(/sbin/ip addr show dev eth0 | sed -nr 's/.*inet ([0-9.]+).*/\1/p');
+EOF
+
+    set +x
+    cat >> $SCRIPT_FILE << EOF
+set +x
+SCAL_PASS=$SCAL_PASS
+RING_VERSION=$RING_VERSION
+set -x
+
+sudo aptitude install -y git
+git clone $OS_CI_REPOSITORY
+git -C openstack-ci-scripts checkout $JOB_GIT_REVISION
+. openstack-ci-scripts/jenkins/ring-install.sh
+initialize
+add_source
+install_base_scality_node
+install_supervisor
+install_ringsh
+build_ring
+show_ring_status
+install_sfused
+
+sudo apt-get -y install nfs-common
+sudo sed -i 's#"type": "fuse"#"type": "nfs"#' /etc/sfused.conf
+sudo sh -c 'echo "/ 127.0.0.1(rw,no_root_squash)" > /etc/exports.conf'
+sudo service scality-sfused restart
+
+sudo apt-get install -y python-pbr python-setuptools python-pip git
+sudo pip install git+https://github.com/scality/scality-manila-utils.git
+
+EOF
+    set -x
+
+    chmod a+x $SCRIPT_FILE
```

Comment:
Yes, we could pipe the commands over ssh as done in
https://github.com/scality/openstack-ci-scripts/pull/25/files#diff-7460f2c05eaec9cbd0458323c667fb12R20

This will however expose any passwords in the job log (unless we turn off tracing for the commands executed over ssh). The approach taken here by @cazino will not leak anything to the logs.